javacord-api/src/main/java/org/javacord/api/entity/server/Server.java

```diff
@@ -2282,6 +2283,55 @@ default CompletableFuture<Void> banUser(long userId, Duration duration, String r
         return banUser(Long.toUnsignedString(userId), duration, reason);
     }
 
+    /**
+     * Bans the given user from the server.
```

Comment:
Change this and the Javadoc from the other newly added methods
```suggestion
     * Bans the given users from the server.
```